src/styles/components/generic/_grid.scss

```
@@ -0,0 +1,71 @@
+        }
+    }
+}
+
+@mixin make-cols($prefix: false) {
+    $align: (
+        left: start,
+        center: center,
+        right: end
+    );
+
+$col: "col-";
```

Stylelint (17.12.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)